app/assets/stylesheets/thredded/components/_topic-header.scss

```diff
@@ -60,3 +60,67 @@
   font-size: $thredded-font-size-small;
   color: $thredded-secondary-text-color;
 }
+
+
+@mixin following {
+  &::after {
+    display: inline-block;
+    font-size: 1em;
+    width: 1.4rem;
+    height: 1.4rem;
+    position: absolute;
+    top: 0;
+    right: -1.6rem;
+    background-image: image-url("thredded/follow.svg");
```

Comment:
You could render a single inline SVG with the icon, and then an svg with `<use>` that references the icon, and style the colors with CSS. http://tympanus.net/codrops/2015/07/16/styling-svg-use-content-css/